File: Examples/Android/aspose-cloud-words-android/src/main/java/com/aspose/words/model/ParagraphLinkCollectionResponse.java
package com.aspose.words.model;

import com.aspose.words.model.ParagraphLinkCollection;
public class ParagraphLinkCollectionResponse {
  private ParagraphLinkCollection Paragraphs = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getParagraphs
	 * Gets ParagraphLinkCollection
	 * @return Paragraphs
	 */
  public ParagraphLinkCollection getParagraphs() {
    return Paragraphs;
  }

	/**
	 * setParagraphs
	 * Sets ParagraphLinkCollection
	 * @param Paragraphs ParagraphLinkCollection
	 */
  public void setParagraphs(ParagraphLinkCollection Paragraphs) {
    this.Paragraphs = Paragraphs;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ParagraphLinkCollectionResponse {\n");
    sb.append("  Paragraphs: ").append(Paragraphs).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words-Cloud-SDK-for-.NET/Aspose.Words.Cloud.Sdk/Model/ParagraphLinkCollectionResponse.cs
namespace Aspose.Words.Cloud.Sdk.Model 
{
  using System;  
  using System.Collections;
  using System.Collections.Generic;
  using System.Runtime.Serialization;
  using System.Text;
  using Newtonsoft.Json;
  using Newtonsoft.Json.Converters;

  /// <summary>
  /// This resonse should be returned by the service when handling:  GET http://api.aspose.com/v1.1/words/Test.doc/paragraphs
  /// </summary>  
  public class ParagraphLinkCollectionResponse : SaaSposeResponse 
  {                       
        /// <summary>
        /// Collection of paragraphs
        /// </summary>  
        public ParagraphLinkCollection Paragraphs { get; set; }

        /// <summary>
        /// Get the string presentation of the object
        /// </summary>
        /// <returns>String presentation of the object</returns>
        public override string ToString()  
        {
          var sb = new StringBuilder();
          sb.Append("class ParagraphLinkCollectionResponse {\n");
          sb.Append("  Paragraphs: ").Append(this.Paragraphs).Append("\n");
          sb.Append("}\n");
          return sb.ToString();
        }
    }
}
